tests/mlp_models.py

```diff
@@ -26,11 +26,16 @@ def dynamic_classifier(
     for layer_size in hidden_layer_sizes:
         hidden = Dense(layer_size, activation="relu")(hidden)
 
+    if compile_kwargs["loss"] == "auto":
+        loss = None
+    else:
+        loss = compile_kwargs["loss"]
```

Comment:
Suggestion:

``` python
loss = None if compile_kwargs["loss"] == "auto" else compile["kwargs"]
```

Reply:
Or should `None` be passed to `build_fn` if `loss="auto"` is specified?

Reply:
I see pros and cons.

Pros:
- Backwards compatibility.

Cons:
- `build_fn` cannot know if `None` was explicitly passed to the constructor or if `"auto"` was converted to None.

I'll add more if I can think of it, and try a quick implementation, it might work just fine...

Reply:
> Suggestion:
> `loss = None if compile_kwargs["loss"] == "auto" else compile["kwargs"]`

Incorporated